commons/src/main/java/org/eclipse/kapua/commons/about/AboutEntry.java

```diff
@@ -29,6 +30,7 @@
             }
 
             APL2 = new License("Apache License 2.0", null, url);
+            EPL =  new License("EPL","Eclipse Public License", url);
```

Comment:
The URL passed into the EPL instance is still the URL of the Apache License.

The correct URL for the EPL would be: `https://www.eclipse.org/legal/epl-v10.html`